connection.go

```diff
@@ -0,0 +1,51 @@
+package srtp
```

Comment:
nice! is this something that can live in `github.com/pions/transport` ? Do you see other subsystems using it (like SCTP) ?